lib/components/PreselectOptions.js

```diff
@@ -0,0 +1,156 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import SelectStep from './SelectScreens';
+
+class PreselectOptions extends PureComponent {
+  constructor(props) {
+    super(props);
+
+    /*
+     * each Opts array builds user selectable options
+     * header   - larger text
+     * text     - sub text
+     * icon     - font awesome icon
+     * click[0] - key in state to change
+     * click[1] - value in state to change
+     *
+     * the click array allows user input
+     * to be propagated to setState where
+     * it can be used to selectively render
+     * sepecific components
+     */
+
+    this.walletTypeOpts = [
```

Comment:
These should be defined outside of the constructor, as they are static and large